.gitignore

```diff
@@ -15,6 +15,7 @@ rerun.txt
 pickle-email-*.html
 *.swp
 kudos.env
+.DS_Store
```

Comment:
See my comment on the other PR about this. Another advantage to configuring the ignore locally is that you won't have to add rules like this on each branch :)